src/index.js

```diff
@@ -156,7 +156,7 @@ async function bootstrapLoop(lastId) {
   }
 
   const res = await npm.findAll({
-    options,
+    ...options,
```

Comment:
why not just pass options?

Reply:
🤦‍♂ 
That's probably what I intended to do first